### PR TITLE
First stab on removing unused code

### PR DESCRIPTION
**Pull request by liamsi**
ref: #206

### CODE REVIEW

File: test/e2e/generator/generate.go [hunk 1]
```diff
@@ -27,8 +27,9 @@ var (
 	}
 
 	// The following specify randomly chosen values for testnet nodes.
+	// TODO settle for one DB only
 	nodeDatabases         = uniformChoice{"goleveldb", "cleveldb", "rocksdb", "boltdb", "badgerdb"}
```
Comment (liamsi) [inline test/e2e/generator/generate.go:31]: What is the preferred / fastest DB again? 
cc @marbar3778 @tzdybal

Comment (marbar3778) [reply test/e2e/generator/generate.go:31]: I would prefer to go with a DB with acid support, Badger is a good candidate for this. We should separate the db work into another PR, that way we can add custom configurations to help optimize for our needs. Ill open an issue.

Comment (liamsi) [reply test/e2e/generator/generate.go]: I've made the e2e tests already only use badger in https://github.com/lazyledger/lazyledger-core/pull/208/commits/4e03ca13d9b36979308b4ac8601c3abf727d2e50

It makes sense to rip out the broader DB story in a separate PR, though. Thanks for opening #211 🙏🏼

**Comment by liamsi** [comment]
I think I got too trigger happy yesterday night and deleted too much already 🙈 🤣

**Comment by liamsi** [comment]
The tests time out and I don't know why exactly.
```
I[2021-03-11|20:05:03.438] Waiting for all nodes to reach height 1018... 
I[2021-03-11|20:05:03.451] Waiting for all nodes to reach height 1035... 
E[2021-03-11|20:05:30.010] timed out waiting for full01 to reach height 1035 
Error: Process completed with exit code 1.
```

Skimming through the logs wasn't insightful either. The only thing I've noticed is that there seem to be more `p2p.ErrRejected` errors than usual. Unclear how this could be related to only keeping builtin abci apps (and removing support for other DBs besides badgersb in e2e tests)

**Comment by liamsi** [comment]
OK, restarting fixed it here. Locally, the tests did not pass but they time out much later if I close all other processes.

**Review by marbar3778 — APPROVED** [review]
LGTM

**Pull request merged**